IntelliTect.TestTools.WindowsTestWrapper/CustomDictionary.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<Dictionary>
+
+  <Words>
+
+    <Recognized>
+
+      <Word>Intelli</Word>
+      <Word>Tect</Word>
+
+    </Recognized>
+
+  </Words>
+
+</Dictionary>
```

Comment:
Hmmm... I thought I did that; there should only be a single file in the solution directory that gets referenced. I'll double check, though.